docs/src/css/tile.module.css

```diff
@@ -0,0 +1,70 @@
+.tile {
+  position: relative;
+  margin: 0.5rem;
+  border-radius: 0.5rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  min-width: 230px;
+  background: var(--card-background);
+  padding: 0.1rem;
+  text-align: center;
+  text-decoration: none;
+  transition: all 0.2s ease-in-out;
+  border: 1px solid var(--card-border);
+  overflow: hidden;
+  text-decoration: none;
+}
+
+.tile h4 {
+  margin-top: 0.5rem;
+  padding: 0 3rem;
```

Comment:
@peintnermax since the text is aligned in the center and boxes have a min width that padding will only cause issues:
![image](https://github.com/zitadel/zitadel/assets/9405495/74960ac0-da6d-4668-a85f-fb0022c4b6f4)